src/CommunityToolkit/Xamarin.CommunityToolkit/Views/RangeSlider/ThumbFrame.shared.cs

```diff
@@ -10,7 +9,7 @@ public ThumbFrame()
 			#region Required work-around to prevent linker from removing the platform-specific implementation
 #if __ANDROID__
 			if (System.DateTime.Now.Ticks < 0)
-				_ = new ThumbFrameRenderer(ToolkitPlatform.Context ?? throw new NullReferenceException());
+				_ = new ThumbFrameRenderer(ToolkitPlatform.Context);
```

Comment:
can you revert this change? Please avoid doing changes that are not related to the issue.
OR is it a rebase issue?

Reply:
I've updated `ToolkitPlatform.Context` and noticed that some calls to it treat it as nullable, while it's not.
Reverted this change